fabric-common/lib/impl/ecdsa/key.js

```diff
@@ -104,11 +104,12 @@ module.exports = class ECDSA_KEY extends Key {
 	/**
 	 * Generates a CSR/PKCS#10 certificate signing request for this key
 	 * @param {string} subjectDN The X500Name for the certificate request in LDAP(RFC 2253) format
+	 * @param {Object[]} extensions Additional X.509v3 extensions for the certificate signing request
```

Comment:
I guess this is an optional parameter? If so, it would be good to mark it optional in the JSDoc by putting `extensions` in square brackets: `[extensions]`

Reply:
Done